docs/architecture/003-namespaced-keywords.md

```diff
@@ -53,3 +56,29 @@ Pros of namespaced keys:
 Cons:
 
 - verbose (mitigated by using short namespaces)
+
+# Amendment to decision
```

Comment:
I would name this amendment specifically and include a date on it too for extra clarity. Let's say `Amendment 6th of March 2019: Same data for same key`

Reply:
done